aws-ecs/packer/ecs-init-weave.dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos
+
+ENV GOPATH=/
+
+# Build amazon-ecs-init
+RUN yum -y install git golang make rpm-build; yum clean all
+RUN mkdir -p /src/github.com/aws/
+RUN git clone -b dev https://github.com/aws/amazon-ecs-init.git /src/github.com/aws/amazon-ecs-init
```

Comment:
Typically the docker containers are immutable, and just contain the dependancies required to build, not the code to build.  That is usually mounted in or clone when the container is run.